COPS-7172 added URL (#4189)
mesosphere/dcos-docs-site: pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/define-control-plane-endpoint/index.md

## Built-in virtual IP

If an external load balancer is not available, use the [built-in virtual IP](../create-cluster/#use-the-built-in-virtual-ip). The virtual IP is _not_ a load balancer; it does not distribute request load among the control plane machines. However, if the machine receiving requests does not respond to them, the virtual IP automatically moves to another machine.

## Single-Node control plane

> A control plane with one node can use its single node as the endpoint, so you will not require an external load balancer, or a built-in virtual IP. However, do not use a single-node control plane configuration in production, as it is not possible to upgrade the control plane if you only have one node. This is due to the fact that CAPI does not perform in-situ upgrades, instead, it replaces machines. If you want to upgrade your cluster, use the default `<code>`--control-plane-replicas`</code>` setting of 3.

Reviewer: ```suggestion
A control plane with one node can use its single node as the endpoint, so you will not require an external load balancer, or a built-in virtual IP. However, do not use a single-node control plane configuration in production, as it is impossible to upgrade the control plane if you only have one node. This is because CAPI does not perform in-situ upgrades, instead, it replaces machines. If you want to upgrade your cluster, use the default `<code>`--control-plane-replicas`</code>` setting of 3.
```
